src/voicereadout.js

```diff
@@ -24,8 +24,11 @@ webOS.voicereadout = {
 	/**
 	 * Read alert text when accessibility VoiceReadout enabled.
 	 * @param {string} s - String to voice readout.
+	 * @param {boolean} c - Clear optoin for TTS. If true it will cut off previous reading. Valid for TV only
 	 */
-	readAlert: function(s) {
+	readAlert: function(s, c) {
+		var clear = (c === undefined)? true : c;
```

Comment:
How about `var clear = (typeof c === 'boolean') ? c : true;`? Sometimes arguments[1] can be null or unexpected types by caller's fault.